database/20200506_create-schema.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE users (
```

Comment:
@mcmoonstruck Pulled most this data out of pg_dump - let me know if anything looks off